Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -147,7 +147,7 @@ function Get-SqlPackageCommandArguments
 
         if($sqlUsername)
         {
-            if($sqlUsername.Contains('@'))
+            if((-not $sqlUsername.Contains("@" + $serverName.Trim())) -and $sqlUsername.Contains('@'))
```

Comment:
This check does not ensure servername string is in the end of sqlusername string. Even if the pattern matches anywhere in between, we will append @.